modules/version-update/pages/update-tool.adoc

```diff
@@ -0,0 +1,516 @@
+= Updating Bonita Runtime 
+:description: This page explains how to update Bonita Runtime version (single or multiple nodes).
+
+This page explains how to update Bonita Runtime version (single or multiple nodes).
+
+[NOTE]
+====
+
+.*Updating* means moving from one Bonita version to a newer one, to benefit from:
+* new features (in Main versions)
+* technology updates (in Main versions)
+* bug fixes (in Maintenance versions)
+ 
+====
+
+== How it works
+
+_Bonita Update Tool_ contains a set of scripts that execute changes on the data for making it compatible with a newer version of Bonita Runtime. 
+The tool is provided as a zip archive that contains an executable script to determine your current version and request the targeted Bonita version for the update.
+
+The update process is step-based, with each maintenance version being equal to a step. +
+The script manages the sequence of steps from your current version to the targeted version.
+After each main or maintenance version update, you have the option to pause or continue to the next step.
+
+=== Can the update tool be used for upgrade/downgrade between community and subscription?
+The answer is *“No”*. 
+The Bonita Update Tool does not help you upgrade/downgrade between the community/subscription versions, so you cannot xref:upgrade-from-community-to-a-subscription-edition.adoc[change edition] at the same time as updating.
+
+=== Bonita version to target
```

Comment:
We should precise that:
* proposed versions are only 2-digit version 7.13, 7.14, ...
* this paragraph invites to update the **Binaries** to the latest maintenance release, whereas the update tool only operates at the **database** level, and for Main releases (2-digit)